locale/fi/servicenavigation.json

```diff
@@ -0,0 +1,70 @@
+    "Komponentille tulee antaa ylätason aria-label, joka kertoo ruudunlukijalle navigaation kontekstin ja tarkoituksen.",
+    "Palvelunavigaatiolle on lisäksi hyvä antaa HTML-id, jotta siihen voidaan liikkua suoraa esimerkiksi SkipLink-navigaatioelementeistä."
+  ],
+  "sections": [
+    {
+      "title": "Mistä komponentti koostuu",
+      "paragraphs": [
+        {
+          "image.key": "",
+          "image.alt": "",
+          "text": "Palvelunavigaatio on navigaatioelementti, jonka sisällä on lista. Komponentin tekninen toteutus mahdollistaa sen, että lista voi sisältää tavanomaisten linkkien lisäksi esimerkiksi painikkeita tai muita semanttisia elementtejä. Kaikki listan elementit ovat kuitenkin semantiikasta huolimatta visuaaliselta tyyliltään samanlaisia. Listan elementit eivät ole toisilleen hierarkisesti alisteisia vaan ne kaikki ovat samalla tasolla."
+        },
+        {
+          "image.key": "",
+          "image.alt": "",
+          "text": "Kapeille ruuduille suunnitellussa variantissaan komponentti sisältää painikkeen, josta varsinainen navigaatioelementti voidaan näyttää ja piilottaa. "
+        }
+      ],
+      "links": [
+        {
+          "text": "",
+          "url": ""
+        }
+      ]
+    },
+    {
+      "title": "Koko ja käyttö",
+      "paragraphs": [
+        {
+          "image.key": "",
+          "image.alt": "",
+          "text": "Navigaatioelementtien sisällä on yllä olevan koodiesimerkin mukaisesti suunniteltu käytettävän RouterLink-komponenttia, joka voidaan semanttisesti renderöidä minä tahansa komponenttina kuten painikkeena tai jonkin kirjaston omana linkkikomponenttina. Tutustu komponentin tekniseen dokumentaatioon saadaksesi lisää esimerkkejä RouterLinkin käytöstä."
+        },
+        {
+          "image.key": "",
+          "image.alt": "",
+          "text": "Palvelunavigaatio asemoidaan tavallisesti sivun vasempaan palstaan, sisältöalueen ollessa sen oikealla puolella. Se ottaa sivuttaissuunnassa kaiken se tilan, joka sille on tarjolla ja täten sen leveyttä kontrolloidaan ympäröivän elementin leveyttä säätämällä. Pienillä ruuduilla (mobiililaitteet) käytetään komponentista niille tarkoitettua varianttia ja asemoidaan komponentti sivun varsinaisen sisältöalueen yläpuolelle. "
+        },
+        {
+          "image.key": "",
+          "image.alt": "",
+          "text": "Palvelunavigaatio soveltuu käyttöön hyvin silloin, kun navigaation linkit ovat hierarkisesti kaikki samalla tasolla ja navigoitava sivusto on tyyliltään sovellusmainen (ks. sähköpostiohjelman tyylinen esimerkki yllä). Sivunavigaatio-komponentti puolestaan on tarkoitettu käytettäväksi silloin, kun navigaatio sisältää alakategorioita. "
```

Comment:
Maybe the last sentence would be clearer if it was arranged a bit differently. Now I had to take a double look at which navigation page I'm looking for. So maybe something like "Jos navigaation on tarpeen sisältää alasivuja- tai kategorioita, tulee käyttää sivunavigaatio-komponenttia."

Reply:
Check!

Reply:
There is still my grammar error that made it to the implementation 😅 instead of "alasivuja- tai kategorioita" it should be "alasivuja tai -kategorioita". So the hyphen is in the wrong spot.

Reply:
Yes... 😅 Fixed it now